.pyre_configuration

```diff
@@ -6,7 +6,7 @@
     ".*/IPython/core/tests/nonascii.*",
     ".*/torchx/examples/apps/compute_world_size/.*"
   ],
-  "site_package_search_strategy": "all",
+  "site_package_search_strategy": "none",
```

Comment:
Because we are passing the site packages directory via `--source-paths` on the commandline, we don't want to use any pyre site package search strategy at all at this time.

Once I fix the bug, we can flip this back and get rid of that commandline flag